sdk/go/common/workspace/plugins_test.go

```diff
@@ -1080,15 +1146,74 @@ func TestBundledPluginSearch(t *testing.T) {
 	err = os.WriteFile(ambientPath, []byte{}, 0o700) //nolint: gosec
 	require.NoError(t, err)
 
+	d := &mockSink{t: t}
```

Comment:
I think for the cases where you don't care about the output, `diagtest.LogSink` may be preferable since it'll log to the testing.T:

https://github.com/pulumi/pulumi/blob/d96533d46a238cc506bb13b6e560637a05e40577/sdk/go/common/testing/diagtest/sink.go#L31

Reply:
! I was looking for something like this
I'll rework things to use that module.